safe/definitions/post_processors/post_processor_functions.py

```diff
@@ -0,0 +1,144 @@
+# coding=utf-8
+# noinspection PyUnresolvedReferences
+from PyQt4.QtCore import QPyNullVariant
+from safe.definitions.hazard_classifications import (
+    not_exposed_class,
+    hazard_classes_all)
+
+__copyright__ = "Copyright 2017, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+# # #
+# Functions
+#
+# Note that the function names and docstrings will be shown in the
+# definitions report, so keep them neat and tidy!
+# # #
+
+
+def multiply(**kwargs):
+    """Simple postprocessor where we multiply the input values.
+
+    :param kwargs: Dictionary of values to multiply
+    :type kwargs: dict
+
+    :return: The result.
+    :rtype: float
+    """
+    result = 1
+    for i in kwargs.values():
+        if isinstance(i, QPyNullVariant) or not i:
+            # If one value is null, we return null.
+            return i
+        result *= i
+    return result
+
+
+def size(size_calculator, geometry):
+    """Simple postprocessor where we compute the size of a feature.
+
+    :param geometry: The geometry.
+    :type geometry: QgsGeometry
+
+    :param size_calculator: The size calculator.
+    :type size_calculator: safe.gis.vector.tools.SizeCalculator
+
+    :return: The size.
+    """
+    feature_size = size_calculator.measure(geometry)
+    return feature_size
+
+
+# This postprocessor function is also used in the aggregation_summary
+def post_processor_affected_function(**kwargs):
+    """Private function used in the affected postprocessor.
+
+    The following are expected in kwargs:
+
+    * classification: The hazard classification to use.
+    * hazard_class: The hazard class to check.
+
+    :return: If this hazard class is affected or not. It can be `not exposed`.
+    :rtype: bool
+    """
+    classification = None
+    for hazard in hazard_classes_all:
+        if hazard['key'] == kwargs['classification']:
+            classification = hazard['classes']
+            break
+
+    for level in classification:
+        if level['key'] == kwargs['hazard_class']:
+            affected = level['affected']
+            break
+    else:
+        affected = not_exposed_class['key']
+
+    return affected
+
+
+def post_processor_population_displacement_function(
+        classification=None, hazard_class=None, population=None):
+    """Private function used in the displacement postprocessor.
+
+    :param classification: The hazard classification to use.
+    :type classification: dict
+
+    :param hazard_class: The hazard class of the feature.
+    :type hazard_class: str
+
+    :param population: We don't use this value here. It's only used for
+        condition for the postprocessor to run.
+    :type population: float, int
+
+    :return: The displacement ratio for a given hazard class.
+    :rtype: float
+    """
+    _ = population
```

Comment:
__[flake8]__
* [F841] local variable '_' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->